modules/apps/portal-scheduler/portal-scheduler-test/src/testIntegration/java/com/liferay/portal/scheduler/internal/test/SchedulerEngineHelperTest.java

```diff
@@ -0,0 +1,411 @@
+							-1, TimeUnit.MINUTE)) {
+
+					_refreshPortalSchedulerBundle();
+
+					try (AutoCloseable autoCloseable2 =
+							_registerIntervalJobConfiguration(
+								16, TimeUnit.MINUTE)) {
+
+						Thread.sleep(3000);
+
+						Assert.assertNotNull(
+							_schedulerEngineHelper.getScheduledJob(
+								TestSchedulerIntervalJobConfiguration.class.
+									getName(),
+								TestSchedulerIntervalJobConfiguration.class.
+									getName(),
+								StorageType.MEMORY_CLUSTERED));
+					}
+				}
+			},
+			new ExpectedLogs(
+				LoggerTestUtil.ERROR, _LOG_CLASS,
+				StringBundler.concat(
+					"Unable to process job. If you want to disable it please ",
+					"blacklist ",
+					TestSchedulerIntervalJobConfiguration.class.getName()),
+				"Interval is either equal or less than 0"));
+	}
+
+	@Test
+	public void testSchedulerGetsActivatedAfterNullCronExpression()
+		throws Exception {
+
+		_assertExpectedLogs(
+			() -> {
+				try (AutoCloseable autoCloseable1 =
+						_registerCronJobConfiguration(null)) {
+
+					_refreshPortalSchedulerBundle();
+
+					try (AutoCloseable autoCloseable2 =
+							_registerCronJobConfiguration("0 0 7 1/2 * ? *")) {
+
+						Thread.sleep(3000);
+
+						Assert.assertNotNull(
+							_schedulerEngineHelper.getScheduledJob(
+								TestSchedulerCronJobConfiguration.class.
+									getName(),
+								TestSchedulerCronJobConfiguration.class.
+									getName(),
+								StorageType.MEMORY_CLUSTERED));
+					}
+				}
+			},
+			new ExpectedLogs(
+				LoggerTestUtil.ERROR, _LOG_CLASS,
+				StringBundler.concat(
+					"Unable to process job. If you want to disable it please ",
+					"blacklist ",
+					TestSchedulerCronJobConfiguration.class.getName()),
+				"Cron expression is null or empty"));
+	}
+
+	private void _assertExpectedLogs(
+			UnsafeRunnable<Exception> runnable, ExpectedLogs expectedLogs)
+		throws Exception {
+
+		try (LogCapture logCapture = LoggerTestUtil.configureLog4JLogger(
+				expectedLogs._loggerClassName, expectedLogs._level)) {
+
+			runnable.run();
+
+			List<LogEntry> logEntries = logCapture.getLogEntries();
+
+			for (LogEntry logEntry : logEntries) {
+				Assert.assertEquals(
+					expectedLogs._level, logEntry.getPriority());
+
+				Assert.assertTrue(
+					ArrayUtil.contains(
+						expectedLogs._logMessages, logEntry.getMessage()));
+
+				Throwable throwable = logEntry.getThrowable();
+
+				Assert.assertTrue(
+					ArrayUtil.contains(
+						expectedLogs._logMessages, throwable.getMessage()));
+			}
+		}
+	}
+
+	private void _refreshPortalSchedulerBundle() throws Exception {
+		for (Bundle bundle : _bundleContext.getBundles()) {
+			String symbolicName = bundle.getSymbolicName();
+
+			if (symbolicName.equals("com.liferay.portal.scheduler")) {
+				Bundle frameworkBundle = _bundleContext.getBundle(0);
+
+				FrameworkWiring frameworkWiring = frameworkBundle.adapt(
+					FrameworkWiring.class);
+
+				final DefaultNoticeableFuture<FrameworkEvent>
```

Comment:
@marianoalvarosaiz No need to declare `final` here, it can be implied as final because there's no other modification.